modules/services/repositories/src/main/java/au/com/shiftyjelly/pocketcasts/repositories/download/UpdateEpisodeDetailsTask.kt

```diff
@@ -45,7 +46,9 @@ class UpdateEpisodeDetailsTask @AssistedInject constructor(
             Timber.i("Downloading Meta Data for ${episodeUuids.size} episodes")
 
             for (episodeUuid in episodeUuids) {
-                val episode = episodeManager.findByUuid(episodeUuid) ?: continue
+                val episode = runBlocking {
+                    episodeManager.findByUuid(episodeUuid)
+                } ?: continue
```

Comment:
```suggestion
                val episode = episodeManager.findByUuid(episodeUuid) ?: continue
```
The `doWork` method is a suspend function so this might not need a runBlocking.